### ISSUE TITLE
ariaAOIassist suggestions

### ISSUE THREAD

**Issue by rzinke** [aria-tools/ARIA-tools#139]
I don't find the red highlighting of "invalid frames" very helpful, and it is often distracting. Perhaps we could remove this feature.

It would also be nice to have an option to exclude dates from the final outputs that remain in the original csv file, but that I intend to exclude. That way, I can have a clean estimate of the outputs. 

Let me know if these are of interest to anyone else.

**Comment by dbekaert** [comment]
Agreed on the red frames can be removed.

Perhaps more instructive would be to color individual date segments?
Shows more clearly on a date if there is a spatial gab?
Might not be as easy if we only have bbox center coordinates.

Including excldue dates would remvoe them from control plots. I think it works for user friendly-ness. For ARIA would recommend to eitehr grey-scale them as we then can still use the plots for verification of production?

**Comment by rzinke** [comment]
_Agreed on the red frames can be removed._
I can make the modifications.

_Perhaps more instructive would be to color individual date segments? Shows more clearly on a date if there is a spatial gab? Might not be as easy if we only have bbox center coordinates._
Not sure exactly what you mean. Could you explain?

_Including excldue dates would remvoe them from control plots. I think it works for user friendly-ness. For ARIA would recommend to eitehr grey-scale them as we then can still use the plots for verification of production?_
Greying them out would be good. What I really need this for is to more easily create the AOI's when I refine them in Google Earth. All the irrelevant frames getting in the way makes precise AOI selection harder.

**Comment by dbekaert** [comment]
so this if for a single date. if the date is broken into different spatial subset due to e.g. a missing frame then color-code in differnt colors. that will make it visually very apparent there is a gap, compared to all black markers.

Could grey them in the marker plot, and not generate a kmz for them?

**Comment by rzinke** [comment]
Those sound like good updates. I will start work on them.

**Comment by rzinke** [comment]
I have made the suggested modifications to the ariaAOIassist function. It now has improved filtering capabilities, including the ability to filter by dates. I also improved the functionality of highlighting dates for which coverage is incomplete.

To assist the user with drawing the precise AOI in Google Earth, I added the function to automatically exclude dates that do not fill the entire area from the kml output. 

The routines are now encoded using the Python class structure, and data manipulation leverages the Pandas dataframe.
